packages/terra-functional-testing/lib/docker/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.7'
+
+services:
+  hub:
+   image: selenium/hub:3.141.59-zirconium
+   ports:
+     - "4444:4444"
+
+  chrome:
+    image: selenium/node-chrome:3.141.59-zirconium
+    volumes:
+      - /dev/shm:/dev/shm
```

Comment:
Could you add comments about why we're volume mounting?

Reply:
I'll add some comment, this file is nearly a direct copy past from the selenium-docker recommendation:

https://github.com/SeleniumHQ/docker-selenium#version-3-with-swarm-support